dev/io.openliberty.microprofile.openapi.4.0.internal.services/src/io/openliberty/microprofile/openapi40/internal/services/impl/package-info.java

```diff
@@ -1,13 +1,15 @@
 /*******************************************************************************
- * Copyright (c) 2024, 2025 IBM Corporation and others.
+ * Copyright (c) 2025 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License 2.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-2.0/
  *
  * SPDX-License-Identifier: EPL-2.0
  *******************************************************************************/
-@com.ibm.websphere.ras.annotation.TraceOptions(traceGroup = Constants.TRACE_GROUP, messageBundle = Constants.TRACE_OPENAPI)
+@TraceOptions(traceGroup = Constants.TRACE_GROUP)
 package io.openliberty.microprofile.openapi40.internal.services.impl;
```

Comment:
This slightly odd change is because the original package was refactored into a different project and a new package created with the same name.